cql3/prepared_statements_cache.hh

```diff
@@ -106,7 +106,7 @@ public:
 
 private:
     using cache_key_type = typename prepared_cache_key_type::cache_key_type;
-    // Keep the entry in the "new generation" partition till 2 hits because
+    // Keep the entry in the "unprivileged" cache section till 2 hits because
```

Comment:
I don't remember ever asking to rename the "new generation" with "unprivileged". I actually think it's a worse name.

I do prefer the name "section" instead of "partition".

Reply:
https://github.com/scylladb/scylla/pull/9398#discussion_r718589824

And it was me who suggested it. The "privileged" comes from the fact that entries in this section are going to be evicted only after there are no entries in the "unprivileged" section to evict.

Reply:
Ok. Your commit message suggests these are changes which asked for, yet a big chunk of the changes are renaming "new generation" to "unprivileged" which I never asked for, so it was confusing :-)

Reply:
The series contains not *only* changes that you've requested. In particular there is a bug fix (the last patch) that wasn't part of v1 at all.

Reply:
But if you referred THIS patch comment then, yeah, it contains fixes addressing your comments and a particular change has been communicated to you in v1 and you didn't object, so... ;)